src/postgres/inventory/paginator/PgPaginatorOrderingAttributes.ts

```diff
@@ -99,7 +102,7 @@ implements Paginator.Ordering<TInput, PgClassType.Value, AttributesCursor> {
       ${_offset != null ? sql.query`offset ${sql.value(_offset)}` : sql.query``}
     `)
 
-    let { rows } = await client.query(query)
+    const { rows } = await client.query(query)
```

Comment:
(Incidentally the assigning lower down is unnecessary, `Array.prototype.reverse` is an in-place modification